src/map/GMapPinLine.js

```diff
@@ -16,6 +16,9 @@
             .on("click", function (row, col, sel) {
                 context.click(context.rowToObj(row.ext.origRow), "", sel);
             })
+            .on("dblclick", function (row, col, sel) {
```

Comment:
Not consistent naming -- some of the functions in this pull have (row, col, sel) some have (row, column, selected)